spec/components/edition/show/actions_component_spec.rb

```diff
@@ -134,7 +134,7 @@
 
   describe "link to delete the edition" do
     (Edition.new.available_states - %i[published superseded deleted]).each do |state|
-      it "should appear for draft editions" do
+      it "should appear for 'in-progress' editions" do
```

Comment:
Is it worth taking your commit message and putting it into the documentation / code in some form? This can be done as a new jira task IMO, but we should make a ticket if so.

Reply:
yes good point, let's have a think?